routes/api/users.js

```diff
@@ -1,15 +1,40 @@
 import express from 'express';
+import multer from 'multer';
+import path from 'path';
 
+import verifyJWT from '../../server/middlewares/verifyJWT';
 import UserValidation from '../../server/middlewares/UserValidation';
 import UserController from '../../server/controllers/UsersController';
 import facebookPassportRoutes from '../../server/config/facebookPassportRoutes';
 
 const {
   validateUserSignUp,
   checkExistingEmail,
-  validateUserLogin
+  validateUserLogin,
 } = UserValidation;
-const { userLogin, signUp } = UserController;
+
+const {
+  userLogin,
+  signUp,
+  updateProfile,
+} = UserController;
+
+const upload = multer({
```

Comment:
Why have you included this in this file? This _multer instance_ could be in your helper/middleware folder and then imported into this file so you can use on the route for user profile update. Another thing this helps you do is that it makes it reusable in some other part of your codebase.